libs/blocks/gnav/gnav.js

```diff
@@ -155,7 +155,6 @@ class Gnav {
 
   decorateLogo = () => {
     const logo = this.body.querySelector('.adobe-logo a');
-    if (!logo) return null;
```

Comment:
Why is this being deleted?

Reply:
not sure how that made it in the commit; removed!!